examples/src/content/jcr_root/apps/core-components-examples/clientlibs/clientlib-site/styles/components/demo.less

```diff
@@ -125,7 +125,7 @@
     }
 
     .cmp-examples-demo__property-value {
-        color: @cmp-examples-color-green-600;
+        color: @cmp-examples-color-red-700;
```

Comment:
Here we can use `@cmp-examples-color-green-700` as we do in the Google Prettify theme.

Reply:
@richardhand I tried that but that did not pass the audit hence I switched to red-700

Reply:
Strange, chrome says it's 4.92 (threshold 4.5) for my setup (Chrome, Mac).

<img width="227" alt="screenshot 2019-02-28 at 09 39 19" src="https://user-images.githubusercontent.com/25616660/53554744-be68dd80-3b40-11e9-8e73-6a678b359073.png">